src/CLI.hs

```diff
@@ -139,6 +139,9 @@ main = do
 
   -- We may need custom builds with mode
   let depManifestPath = cwd </> "component.json"
+
+  -- TODO(Yorkie): to be remove if verified by @Kenneth
```

Comment:
Okay, will remove this TODO, actually i have used the dynamic dependencies in `Script.hs`